### PR TITLE
Add a file for Apple Valley, MN

### PR DESCRIPTION
**Pull request by dave-burke**
This pull request is to add a hometown file for Apple Valley, MN :apple: 

Some :question: to answer:
1. If you've been to Apple Valley, what did you do? What did you think of it?
1. If you have kids, what kind of things do they do?
1. What are some neat places most people don't know about?

### CODE REVIEW

File: dave-burke-av.md [hunk 1]
```diff
@@ -0,0 +1,8 @@
+# Apple Valley
+
+### Restaurants
+
+### Activities
```
Comment (githubteacher) [inline dave-burke-av.md:14]: @dave-burke what's the weather like in 🍎 valley?

**Comment by githubteacher** [comment]
> If you've been to Apple Valley

I haven't!

**Comment by dave-burke** [comment]
This pull request resolves issue #9

File: dave-burke-av.md [hunk 2]
```diff
@@ -0,0 +1,13 @@
+# Apple Valley
+
+### Weather
+
+- Hot in the summer :snowflake:
+- Cold in the winter :sun_with_face:
+
+### Restaurants
```
Comment (githubteacher) [inline dave-burke-av.md:8]: If you add some restaurants I think this will be ready to 🚢 :shipit:

**Review by githubteacher — CHANGES_REQUESTED** [review]

**Review by githubteacher — APPROVED** [review]

**Pull request merged**